code/modules/library/lib_machines.dm

```diff
@@ -161,6 +162,7 @@ GLOBAL_LIST(cachedbooks) // List of our cached book datums
 // It's December 25th, 2014, and this is STILL here, and it's STILL relevant. Kill me
 /obj/machinery/computer/libraryconsole/bookmanagement
 	name = "book inventory management console"
+	desc = "Librarian's command station."

```

Comment:
*The* librarian's

Reply:
I don't know what the grammatical term is for this but it makes sense if you think about it from a kind of matter-of-fact perspective. I can change it if you really care.

Reply:
curator*

Reply:
The person running a library is still called a librarian

Reply:
no, they're the curator now

Reply:
I meant that in a general sense, not referring to the job of the ss13 crew